Sources/BraveWallet/WalletStrings.swift

```diff
@@ -2654,5 +2654,47 @@ extension Strings {
       value: "Buy not supported for selected network",
       comment: "A placeholder in Buy Screen, when user switched to a network that Brave currently doesn't support buy token with Wyre."
     )
+    public static let auroraBridgeAlertTitle = NSLocalizedString(
+      "wallet.auroraBridgeAlertTitle",
+      tableName: "BraveWallet",
+      bundle: .strings,
+      value: "Open the Rainbow Bridge App?",
+      comment: "The title of the alert that will show up when users click on `Bridge to Aurora` button inside asset details screen for tokens that support briding to `Aurora`."
+    )
+    public static let auroraBridgeAlertDescription = NSLocalizedString(
+      "wallet.auroraBridgeAlertDescription",
+      tableName: "BraveWallet",
+      bundle: .strings,
+      value: "Rainbow Bridge is an independent service that helps you bridge assets across networks, and use your crypto on other networks and DApp ecosystems. Bridging assets to other networks has some risks.",
```

Comment:
`DApp ecosystems` is this DApp spelled like this on purpose? with two capital letters

Reply:
I've seen `DApp` on internet, but tbh i've all versions like `dApp`, `Dapp`, or `dapp`. This copy is confirmed by Ryan McFadden.